nomad/structs/structs.go

```diff
@@ -4945,6 +4945,9 @@ type Namespace struct {
 	// against.
 	Quota string
 
+	// Capabilities is the set of capabilities allowed for this namespace
+	Capabilities *NamespaceCapabilities
```

Comment:
Note that we'll need to add handling of this to the `Copy` and `SetHash` methods as well.